README.md

```diff
@@ -46,20 +46,26 @@ for (let i = 0; i < qualityLevels.length; i++) {
 let currentSelectedQualityLevelIndex = qualityLevels.selectedIndex;
 ```
 
-### HLS
+### Populating the list
+Initially the list of quality levels will be empty. You can add quality levels to the list by using `QualityLevelList.addQualityLevel` for each quality level specific to your source. `QualityLevelList.addQualityLevel` takes in a `Representation` object (or generic object with the require properties). All properties except are required except `width` and `height`.
```

Comment:
"the require properties). All properties except are required except" => "the required properties). All properties are required except"